corehq/apps/app_manager/views/releases.py

```diff
@@ -125,7 +125,10 @@ def _get_batch(start_build=None, skip=None):
             app_es = app_es.is_released()
         if query:
             app_es = app_es.add_query(build_comment(query), queries.SHOULD)
-            app_es = app_es.add_query(version(query), queries.SHOULD)
+            try:
+                app_es = app_es.add_query(version(int(query)), queries.SHOULD)
+            except ValueError:
```

Comment:
What was happening here? I swear I tested out text searches when implementing the version search...

Reply:
https://sentry.io/organizations/dimagi/issues/840150183